### PR TITLE
Connected the search function to the main page

### PR DESCRIPTION
**Pull request by Alikpld**
the "test" search function is now working on the main page, I limited the number of the input characters as well. I changed some functions and changed the style a little bit.  we can get rid of the /db page now. Next step is to integrate search with the add product. I'll create a branch based on this for the integration

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,18 +21,19 @@ express()
     try {
       const client = await pool.connect();
       const result = await client.query(`SELECT prod_name, type, exp_dt, qty FROM fridge_products NATURAL JOIN products`);
-      const results = { 'results': (result) ? result.rows : null};
-      res.render('pages/index', results );
+      const searchresult = await client.query(`SELECT * FROM products where upper(prod_name)=upper('${req.query.searchParam}')`);
```
Comment (nmdoran) [inline index.js]: Filter by user_ID IN ('0', '1')

File: index.js [hunk 2]
```diff
@@ -21,18 +21,19 @@ express()
...
       console.error(err);
       res.send("Error " + err);
     }
 })
-  .get('/db', jsonParser, async (req, res) => {
+  .get('/searchProduct', jsonParser, async (req, res) => {
     try {
       const client = await pool.connect();
-      const result = await client.query(`SELECT * FROM products where prod_name='${req.query.searchParam}'`);
+      const result = await client.query(`SELECT * FROM products where upper(prod_name)=upper('${req.query.searchParam}')`);
```
Comment (nmdoran) [inline index.js]: Filter by user_ID IN ('0', '1')

File: public/client.js [hunk 3]
```diff
@@ -30,4 +30,12 @@ function removeProduct() {
...
\ No newline at end of file
+}
+
+/*function searchProduct() {
+  console.log("searching a product...")
+  const userRequest = new XMLHttpRequest();
+  userRequest.open('post', '/db');
+  userRequest.setRequestHeader("Content-Type", "application/json;charset=UTF-8")
+  userRequest.send(JSON.stringify({'product':document.getElementById("addProduct").value}));
+/*
```
Comment (mdschnei) [inline public/client.js]: Minor hiccup here, as comment blocks should be opened with **/***, but closed with ***/**. As a result, here we have two opening comment block symbols, so please change the last one on line 41 to */.

File: index.js (hunk 1 above)
Comment (Alikpld) [reply index.js]: updated the query! Thanks Nicole

File: public/client.js (hunk 3 above)
Comment (Alikpld) [reply public/client.js]: my bad, i updated it :) Thanks for catching it

**Pull request merged**